src/app/models/AttributeDecorators.ts

```diff
@@ -2,23 +2,42 @@ import { Id, Ids, ImageSizes, ImageUrl } from "@interfaces/apiInterfaces";
 import { DateTime, Duration } from "luxon";
 import { AbstractModel } from "./AbstractModel";
 
+function persistAttr(model: AbstractModel, key: string, opts: boolean | BawPersistAttributeOptions) {
+    model[AbstractModel.createAttributesKey] ??= [];
+    model[AbstractModel.updateAttributesKey] ??= [];
```

Comment:
could this be a static/prototypical property?
Or else done on initialization in constructor?

Reply:
Is it not already statically typed?

Reply:
Static as in an attribute of the class/type/prototype and not the instance

Reply:
Will be part of #1005